### ISSUE TITLE
WebExtension support

### ISSUE THREAD

**Issue by snkedive** [km3r/Pass2PolyRatings#1]
Firefox version found [here](https://addons.mozilla.org/en-US/firefox/addon/pass2polyratings/) is not compatible with Firefox 57+.

**Comment by km3r** [comment]
Yeah that never quite worked. Use the chrome version or feel free to port
it yourself.
